modules/Context.js

```diff
@@ -135,6 +135,10 @@ export class Context extends EventEmitter {
       return true;  // map.editableDataEnabled();  // todo: disallow editing if OSM layer is off
     };
 
+    // ColorSystem
+    const colors = this.systems.colors;
```

Comment:
Ahh, okay. So since that `Promise.resolve()` call includes `allSystems`, which has the color system in it, I shouldn't need to add anything at all here right?